prisma/migrations/20240509184422_adds_index_for_resolve_workflow_run_status_and_list_job_runs_for_workflow_run/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- CreateIndex
+CREATE INDEX CONCURRENTLY "JobRun_workflowRunId_tenantId_idx" ON "JobRun"("workflowRunId", "tenantId");
```

Comment:
Let's collapse into a single migration - migrations should correspond to releases, so `v0.26.0` would work for this. You can do this manually as well by copying them all into a migration folder called `20240509163031_v0_26_0` or something.